src/locales/en-us/messages.json

```diff
@@ -600,17 +600,17 @@
     "component": "WordUsageExamplesFilters"
   },
   "WORDUSAGE_FILTERS_TYPE_FILTER_CURRENT_RESULTS": {
-    "message": "Filter these results by Author and/or Work",
+    "message": "Limit these results by Author and/or Work",
     "description": "Word Usage Examples Type filter",
     "component": "WordUsageExamplesFilters"
   },
   "WORDUSAGE_FILTERS_AUTHOR_CLEAR": {
-    "message": "Clear author filter",
+    "message": "Clear author limit",
     "description": "Word Usage Examples Type filter",
     "component": "WordUsageExamplesFilters"
   },
   "WORDUSAGE_FILTERS_TEXTWORK_CLEAR": {
-    "message": "Clear textwork filter",
+    "message": "Clear textwork limit",
```

Comment:
please change "textwork" to "work" here too. thanks!

Reply:
Oh, missed it - fixed